### ISSUE TITLE
OverFlow Error

### ISSUE THREAD

**Issue by bquinn17** [MARIE-js/MARIE.js#272]
I was wondering if I could get some more information on issue #169 that was closed before, and why it was decided to be handled that way. The value `FFFF` is equivalent to `65535` in decimal, not `32767` so I'm not sure why the error is thrown at that value. Also, the original MARIE implementation would not throw an error when the value `FFFF` was passed. It would simply wrap around to a negative number, and continue to be incremented from there. This is also the same way overflows of primitive types work in most programming languages.

**Comment by bquinn17** [comment]
I just realized the value `32767` is used because you're using a signed decimal representation.

**Comment by lorderikir** [comment]
Hi,

Thanks for submitting an issue!

While, we do agree that we shouldn't handle Overflow Errors. We have encountered with end-users that when it goes beyond the range `-32767` to `32767`, it slows down their browser and may cause crashes (especially on low end devices), to make it more crash 'gracefully' we decided to make it throw an error when it goes beyond the range. 

If you have have a better solution, feel free to fork this project and make a Pull Request.
